### ISSUE TITLE
How to modify the output layer to only give the next character in sequence given the previous characters ? 

### ISSUE THREAD

**Issue by jjsr** [sherjilozair/char-rnn-tensorflow#142]
I am new to NNs .I am able to understand the model is trained in a way that given input character sequence it can predict next character , Please guide me what model parameters i have to save while training or any pseudo  algorithm to implement the same  will help

**Comment by victorr0** [comment]
Could you rephrase your question, because it is unclear / ambiguous. The char-rnn already does what you ask in the title of your question. Couldn't hurt to check out the README.md, _Tuning_-section may be helpful for model parameters.

**Comment by jjsr** [comment]
Thanks a lot victorr0 for writing back . Sorry for my bad phrasing . Yes I am able to figure that out later since i am quite new in the domain. Thanks for sharing this wonderful repo . that saves my life
